src/Ethereum/ABI/Parameters.cpp

```diff
@@ -59,7 +59,11 @@ std::string ParamSet::getType() const {
     std::string t = "(";
     int cnt = 0;
     for (auto p : _params) {
-        if (cnt++ > 0) {
+        bool isFirst = cnt++ > 0;
```

Comment:
Hey @matsjj , I can't undo this as we need a small change here because when `p->shouldSkipEncodeInFunctionSignature` we should still increment the `cnt` but we should not add a comma `,`.
Maybe you have a better formatting in mind?